File: nowa/2017/python/main.py
import zad6_1
import zad6_2
import zad6_3
import zad6_4

# zebranie odpowiedzi z podpunktow i zapisanie ich do pliku
result = '\n'.join([zad6_1.answer, zad6_2.answer,
                    zad6_3.answer, zad6_4.answer])
with open('wyniki.txt', mode='w') as f:
    f.write(result)

File: nowa/2017/c++/main.cpp
#include "zad6_1.cpp"
#include "zad6_2.cpp"
#include "zad6_3.cpp"
#include "zad6_4.cpp"
#include <fstream>
using namespace std;

int main()
{
    cout << zad6_1() << endl
         << zad6_2() << endl
         << zad6_3() << endl
         << zad6_4() << endl;

    ofstream file("wyniki.txt");

    file << zad6_1() << endl
         << zad6_2() << endl
         << zad6_3() << endl
         << zad6_4() << endl;
}